include/madrona/mw_gpu.hpp

```diff
@@ -45,6 +45,16 @@ struct StateConfig {
 
     // Number of exported ECS components
     uint32_t numExportedBuffers;
+
+    // This is required if the BVH is to be invoked (holds object information).
+    imp::ImportedAssets::GPUGeometryData *geometryData;
+
+    Span<const imp::SourceMaterial> materials;
```

Comment:
I'm not a big fan of this, the import namespace is intended to represent source assets that then get preprocessed into some final runtime form. This might not need to be changed for merge though.

Reply:
this is a mistake - not supposed to be there.